src/views/SafetyStock.vue

```diff
@@ -10,6 +10,7 @@
     <ion-content>
       <main v-if="ruleGroup.ruleGroupId && rules.length">
```

Comment:
Add handling for displaying archived card if archived rules are present, in current case if we have archived rules but do not have active rules then the archived card will not be displayed.